lib/executor/ramping_arrival_rate_test.go

```diff
@@ -243,6 +243,60 @@ func TestRampingArrivalRateRunCorrectRateWithSlowRate(t *testing.T) {
 	assert.Equal(t, int64(2), es.GetInitializedVUsCount())
 }
 
+func BenchmarkRampingArrivalRateRun(b *testing.B) {
+	tests := []struct {
+		prealloc null.Int
+	}{
+		{prealloc: null.IntFrom(100)},
+		{prealloc: null.IntFrom(1000)},
+	}
+
+	for _, tc := range tests {
+		b.Run(fmt.Sprintf("PreAllocatedVUs%d", tc.prealloc.ValueOrZero()), func(b *testing.B) {
+			engineOut := make(chan stats.SampleContainer, 100)
+			defer close(engineOut)
+
+			es := lib.NewExecutionState(lib.Options{}, mustNewExecutionTuple(nil, nil), 1, uint64(tc.prealloc.ValueOrZero()))
```

Comment:
```suggestion
			es := lib.NewExecutionState(lib.Options{}, mustNewExecutionTuple(nil, nil), uint64(tc.prealloc.Int64), uint64(tc.prealloc.Int64))
```

Though for some reason I get a panic